ui/app/pages/keychains/index.scss

```
@@ -61,134 +61,171 @@
-  top: -25px;
-}
+.import-account {
+  &__title {
+    color: #1b344d;
+    font-size: 40px;
+    line-height: 51px;
+    margin-bottom: 10px;
+  }
 
-.import-account__secret-phrase {
-  height: 190px;
-  width: 495px;
-  border: 1px solid #cdcdcd;
-  border-radius: 6px;
-  background-color: #fff;
-  padding: 17px;
-  font-size: 16px;
-}
+  &__back-button {
+    margin-bottom: 18px;
+    color: #22232c;
+    font-size: 16px;
+    line-height: 21px;
+    position: absolute;
+    top: -25px;
+  }
 
-.import-account__secret-phrase::placeholder {
-  color: #9b9b9b;
-  font-weight: 200;
-}
+  &__secret-phrase {
+    height: 190px;
+    width: 495px;
+    border: 1px solid #cdcdcd;
+    border-radius: 6px;
+    background-color: #fff;
+    padding: 17px;
+    font-size: 16px;
+  }
 
-.import-account__faq-link {
-  font-size: 18px;
-  line-height: 23px;
-}
+  &__secret-phrase::placeholder {
+    color: #9b9b9b;
+    font-weight: 200;
+  }
 
-.import-account__selector-label {
-  color: #1b344d;
-  font-size: 16px;
-}
+  &__faq-link {
+    font-size: 18px;
+    line-height: 23px;
+  }
 
-.import-account__dropdown {
-  width: 325px;
-  border: 1px solid #cdcdcd;
-  border-radius: 4px;
-  background-color: #fff;
-  margin-top: 14px;
-  color: #5b5d67;
-  font-size: 18px;
-  line-height: 23px;
-  padding: 14px 21px;
-  appearance: none;
-  -webkit-appearance: none;
-  -moz-appearance: none;
-  cursor: pointer;
-}
+  &__selector-label {
+    color: #1b344d;
+    font-size: 16px;
+  }
 
-.import-account__description-text {
-  color: #757575;
-  font-size: 18px;
-  line-height: 23px;
-  margin-top: 21px;
-}
+  &__dropdown {
+    width: 325px;
+    border: 1px solid #cdcdcd;
+    border-radius: 4px;
+    background-color: #fff;
+    margin-top: 14px;
+    color: #5b5d67;
+    font-size: 18px;
+    line-height: 23px;
+    padding: 14px 21px;
+    appearance: none;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    cursor: pointer;
+  }
 
-.import-account__input-wrapper {
-  display: flex;
-  flex-flow: column nowrap;
-  margin-top: 30px;
-}
+  &__description-text {
+    color: #757575;
+    font-size: 18px;
+    line-height: 23px;
+    margin-top: 21px;
+  }
 
-.import-account__input-error-message {
-  margin-top: 10px;
-  width: 422px;
-  color: #ff001f;
-  font-size: 16px;
-  line-height: 21px;
-}
+  &__input-wrapper {
+    display: flex;
+    flex-flow: column nowrap;
+    margin-top: 30px;
+  }
 
-.import-account__input-label {
-  margin-bottom: 9px;
-  color: #1b344d;
-  font-size: 18px;
-  line-height: 23px;
-}
+  &__input-error-message {
+    margin-top: 10px;
+    width: 422px;
+    color: #ff001f;
+    font-size: 16px;
+    line-height: 21px;
+  }
 
-.import-account__input-label__disabled {
-  opacity: 0.5;
-}
+  &__input-label {
+    margin-bottom: 9px;
+    color: #1b344d;
+    font-size: 18px;
+    line-height: 23px;
+  }
 
-.import-account__input {
-  width: 350px;
-}
+  &__input-label__disabled {
+    opacity: 0.5;
+  }
```

While we're here, we should do this and change the corresponding classname in the file:
```suggestion

    &--disabled {
      opacity: 0.5;
    }
  }  
```

---

ISTM that this class isn't being used in either of the vault files :thinking: